examples/todos-postgres/src/main.rs

```diff
@@ -0,0 +1,111 @@
+use sqlx::PgPool;
+use std::env;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+struct Args {
+    #[structopt(subcommand)]
+    cmd: Option<Command>,
+}
+
+#[derive(StructOpt)]
+enum Command {
+    Add { description: String },
+    Done { id: i64 },
+}
+
+#[async_std::main]
+#[paw::main]
+async fn main(args: Args) -> anyhow::Result<()> {
+    let mut pool = PgPool::new(&env::var("DATABASE_URL")?).await?;
+
+    match args.cmd {
+        Some(Command::Add { description }) => {
+            println!("Adding new todo with description '{}'", &description);
+            println!(
+                "Added new todo with id {}",
+                add_todo(&pool, &description).await.unwrap()
+            );
+        }
+        Some(Command::Done { id }) => {
+            println!("Marking todo {} as done", id);
+            if complete_todo(&pool, id).await.unwrap() {
+                println!("Todo {} is marked as done", id);
+            } else {
+                println!("Invalid id {}", id);
+            }
+        }
+        None => {
+            println!("Printing list of all todos");
+            list_todos(&mut pool).await.unwrap();
+        }
+    }
+
+    pool.close().await;
```

Comment:
I would remove this invocation as the close method is not intended to be called explicitly unless you _need_ to wait on the connections terminating.